CONTRIBUTING.md

```diff
@@ -1,103 +1,118 @@
-# Wandb library
-
-## Get the code / library
-
-Checkout from github:
+# Contributing to `wandb`
+
+We at Weights & Biases ❤️ open source and welcome contributions from the community!
+This guide discusses the development workflow and the internals of the `wandb` client library.
+
+## Development workflow
+
+1. If you are a first-timer, please go to `https://github.com/wandb/client` 
+and click the "Fork" button in the top-right corner of the page. 
+This will create your personal copy of the repository that you will use for development. 
+   - Set up [SSH authentication with GitHub](https://docs.github.com/en/authentication/connecting-to-github-with-ssh).
+   - Clone the forked project to your machine and add the upstream repository 
+     that will point to the main `wandb` project:
+    ```shell
+    git clone https://github.com/<your-username>/client.git
+    cd client
+    git remote add upstream https://github.com/wandb/client.git   
+    ```
+
+2. Develop you contribution.
```

Comment:
Maybe first we point them to the issues page? See if they find the issue if not open a new issue and then contribute a change? Is it an overkill? what do you think? (I think it will help us track the issues)

Reply:
Great idea, added!